pkgs/os-specific/cygwin/w32api/headers.nix

```diff
@@ -0,0 +1,20 @@
+{
+  lib,
+  windows,
+}:
+
+windows.mingw_w64_headers.overrideAttrs (old: {
+  pname = "w32api-headers";
+
+  configureFlags = [ (lib.enableFeature true "w32api") ];
+
+  passthru = old.passthru or { } // {
+    tests = { };
+    incdir = "/include/w32api/";
+    libdir = "/lib/w32api/";
+  };
+
+  meta = old.meta // {
+    maintainers = [ lib.maintainers.corngood ];
```

Comment:
See above for this file too, could couple those within the cygwin pkgset together

Also please add a description block.